internal/plugin.bzl

```diff
@@ -19,9 +19,25 @@ Node.js source files, as well as promoting reuse of plugins across multiple
 postcss_binary targets.
 """
 
+PostcssPluginInfo = provider(fields=["node_require"])
+
+def _postcss_plugin_info_impl(ctx):
+    return [PostcssPluginInfo(node_require=ctx.attr.node_require)]
+
+postcss_plugin_info = rule(
+    implementation = _postcss_plugin_info_impl,
+    attrs = {
+        "node_require": attr.string(
+            default = "",
+            mandatory = True,
+        ),
+    },
+)
+
 def postcss_plugin(
         name,
-        srcs,
+        srcs = [],
+        node_require = "",
```

Comment:
Consider making this `None` so that an error is raised early if it's not defined.

Reply:
Thanks for pointing this out. Maybe it shouldn't be an optional param at all?

Reply:
Changed this to be a required param.